packages/executors/graphql-ws/src/index.ts

```diff
@@ -73,18 +73,20 @@ export function buildGraphQLWSExecutor(
         }
       : WebSocket;
 
-    graphqlWSClient = createClient({
+    const clientOptions: ClientOptions = {
```

Comment:
Why create the clientOptions object and then create a client with them?

Reply:
Leftover :/